airbyte-integrations/connectors/source-orb/source_orb/source.py

```diff
@@ -268,6 +270,13 @@ def transform_record(self, ledger_entry_record):
         del ledger_entry_record["customer"]
         ledger_entry_record["customer_id"] = nested_customer_id
 
+        # Un-nest credit_block -> expiry_date into block_expiry_date and per_unit_cost_basis
+        nested_expiry_date = ledger_entry_record["credit_block"]["expiry_date"]
+        nested_per_unit_cost_basis = ledger_entry_record["credit_block"]["per_unit_cost_basis"]
+        del ledger_entry_record["credit_block"]
+        ledger_entry_record["block_expiry_date"] = nested_expiry_date
+        ledger_entry_record["per_unit_cost_basis"] = nested_per_unit_cost_basis
```

Comment:
```suggestion
        ledger_entry_record["credit_block_expiry_date"] = nested_expiry_date
        ledger_entry_record["credit_block_per_unit_cost_basis"] = nested_per_unit_cost_basis
```

Reply:
@marcosmarxm Thanks for the suggestion! I believe renaming `block_expiry_date` to `credit_block_expiry_date` might cause some issues since we already expose this field on the `ledger_entry_record` and we're just changing where in the response shape we're pulling the data from.

I can rename `per_unit_cost_basis` safely though, since that's a net-new field.